buildspec.yml

```diff
@@ -0,0 +1,75 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: "1.3.0"
+    ACCOUNT: "142577830533"
+    AWS_DEFAULT_REGION: "us-west-2"
+    ECR_REPO: "sagemaker-framework-pull-request-integ"
+    PREPROD_IMAGE: "142577830533.dkr.ecr.us-west-2.amazonaws.com/sagemaker-framework-pull-request-integ"
+    CPU_FRAMEWORK_BINARY: "https://files.pythonhosted.org/packages/71/64/49c5125befd5e0f0e17f115d55cb78080adacbead9d19f253afd0157656a/mxnet-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl"
```

Comment:
is there any point to this anymore or should we just pip install from inside the docker file?

Reply:
If we pip install inside docker, we need to build and push the docker image everytime we update the binary. I feel it's better to just change this variable in buildspec, that's an easier change.

Reply:
isn't the whole point that we are building a fresh image so we can run integration tests against it?

Reply:
@jesterhazy 
I am thinking you want to have this binary downloaded in the codebuild image. Seems I misunderstand you. 

If you mean we should pip install the binary in mxnet dockerfile. Now the binary is passed to dockerfile as an argument hence it is pip installed in dockerfile and we do build a fresh image for test everytime.

Reply:
yes, since we are just installing the normal pypi release of mxnet, why not just call `pip install mxnet` inside the dockerfile?

Reply:
For that part it includes changing framework dockerfile, which should be a framework change. I would like to have a separate story for it. It can be probably combined with upgrade from python 3.5 to 3.6.